src/bench/src/main/scala/osmesa/Bench.scala

```diff
@@ -1,48 +1,48 @@
-package osmesa
+// package osmesa
 
-import java.util.concurrent.TimeUnit
+// import java.util.concurrent.TimeUnit
 
-import scala.util.Try
+// import scala.util.Try
 
-import cats.implicits._
-import org.apache.log4j
-import org.apache.spark._
-import org.apache.spark.sql._
-import org.openjdk.jmh.annotations._
-import osmesa.analytics.oneoffs.Analysis
+// import cats.implicits._
+// import org.apache.log4j
+// import org.apache.spark._
+// import org.apache.spark.sql._
+// import org.openjdk.jmh.annotations._
+// import osmesa.analytics.oneoffs.Analysis
 
-// --- //
+// // --- //
 
-@BenchmarkMode(Array(Mode.AverageTime))
-@OutputTimeUnit(TimeUnit.SECONDS)
-@State(Scope.Thread)
-class Bench {
+// @BenchmarkMode(Array(Mode.AverageTime))
+// @OutputTimeUnit(TimeUnit.SECONDS)
+// @State(Scope.Thread)
+// class Bench {
 
-  var conf: SparkConf = _
-  implicit var ss: SparkSession = _
+//   var conf: SparkConf = _
+//   implicit var ss: SparkSession = _
 
-  @Setup
-  def setup: Unit = {
-    conf = new SparkConf()
-      .setIfMissing("spark.master", "local[*]")
-      .setAppName("road-changes")
-      .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
-      .set("spark.kryo.registrator", classOf[geotrellis.spark.io.kryo.KryoRegistrator].getName)
+//   @Setup
+//   def setup: Unit = {
+//     conf = new SparkConf()
+//       .setIfMissing("spark.master", "local[*]")
+//       .setAppName("road-changes")
+//       .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+//       .set("spark.kryo.registrator", classOf[geotrellis.spark.io.kryo.KryoRegistrator].getName)
 
-    ss = SparkSession.builder.config(conf).enableHiveSupport.getOrCreate
+//     ss = SparkSession.builder.config(conf).enableHiveSupport.getOrCreate
 
-    /* Silence the damn INFO logger */
-    log4j.Logger.getRootLogger().setLevel(log4j.Level.ERROR)
-  }
+//     /* Silence the damn INFO logger */
+//     log4j.Logger.getRootLogger().setLevel(log4j.Level.ERROR)
+//   }
 
-  @TearDown
-  def close: Unit = ss.stop()
+//   @TearDown
+//   def close: Unit = ss.stop()
 
-  @Benchmark
-  def roads: Try[Double] = {
-    val path: String = "/home/colin/code/azavea/vectorpipe/data/isle-of-man.orc"
+//   @Benchmark
+//   def roads: Try[Double] = {
+//     val path: String = "/home/colin/code/azavea/vectorpipe/data/isle-of-man.orc"
 
-    (Try(ss.read.orc(path)) >>= Analysis.newRoadsByUser).map(_.aggregate(0d)({ _ + _._2 }, { _ + _ }))
-  }
+//     (Try(ss.read.orc(path)) >>= Analysis.newRoadsByUser).map(_.aggregate(0d)({ _ + _._2 }, { _ + _ }))
+//   }
 
-}
+// }
```

Comment:
???

Reply:
This benchmark did not compile.  I wanted to keep it for posterity, but I also didn't want to fix it.  If the preference is to kill it, no problem, but it is a good example of benchmarking using spark.